lenskit/algorithms/basic.py

```diff
@@ -260,7 +260,9 @@ def predict_for_user(self, user, items, ratings=None):
         return preds.reindex(items)
 
     def __str__(self):
-        return 'Fallback([{}])'.format(', '.join(self.algorithms))
+
```

Comment:
If we're revising this, drop unnecessary blank line here (I don't think it adds clarity).